docs: dotgo page with demos for attendees (#3021)
sourcegraph/sourcegraph: doc/user/dotgo-eu-2019.md

---
ignoreDisconnectedPageCheck: true
---

# Sourcegraph @ dotGo EU 2019

We loved meeting so many awesome and talented engineers at dotGo EU 2019. The Go community certainly is passionate about developer tools and productivity, and we'll be back next year for sure! - [Quinn](https://twitter.com/sqs), [Loic (left)](https://twitter.com/lguychard) and [Ryan (right)](https://twitter.com/ryan_blunden)

<p class="text-center">
  <img src="https://user-images.githubusercontent.com/133014/55120082-ba819a00-50b1-11e9-8823-6a1fead17a42.jpg" style="width:60%;" />
</p>

> > NOTE: If you weren't able to chat with us at the conference and want guidance for setting up Sourcegraph for your team, head to the [dotGo page](https://about.sourcegraph.com/dotgo) to leave your contact info.

Reviewer: We're likely to remove this URL. To avoid a future broken link, just tell them to email us directly.

>

> > And don't worry, we're not going to spam you, or add you to any mailing lists.

Reviewer: This sentence becomes unnecessary if we tell them to just email us directly.

> The content for the two demos we showed at the conference are below.

Reviewer: ```suggestion
The demos you saw at the conference are below:
```
